lighthouse-core/gather/gatherers/accessibility.js

```diff
@@ -27,10 +27,12 @@ const axe = fs.readFileSync(
 // This is run in the page, not Lighthouse itself.
 /* istanbul ignore next */
 function runA11yChecks() {
-  axe.a11yCheck(document, function(results) {
-    // __returnResults is magically inserted by driver.evaluateAsync
-    __returnResults(results);
+  var fulfill;
+  var result = new Promise(f => {
+    fulfill = f;
   });
+  axe.a11yCheck(document, fulfill);
+  return result;
```

Comment:
Done. We are using single line callback -> Promise conversion in devtools, but with your linting it does not fit a line, so your version works better in your source base.